integration/users/usersinfo_test.go

```diff
@@ -0,0 +1,496 @@
+// Copyright 2021 FerretDB Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package users
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+
+	"github.com/FerretDB/FerretDB/integration"
+	"github.com/FerretDB/FerretDB/integration/setup"
+	"github.com/FerretDB/FerretDB/internal/types"
+	"github.com/FerretDB/FerretDB/internal/util/must"
+	"github.com/FerretDB/FerretDB/internal/util/testutil"
+)
+
+// createUser creates a bson.D command payload to create an user with the given username and password.
+func createUser(username, password string) bson.D {
+	return bson.D{
+		{"createUser", username},
+		{"roles", bson.A{}},
+		{"pwd", password},
+	}
+}
+
+func TestUsersinfo(t *testing.T) {
+	t.Parallel()
+
+	ctx, collection := setup.Setup(t)
+	client := collection.Database().Client()
+
+	// Map of users per database suffix
```

Comment:
```suggestion
	// Map of users per database suffix.
```
(full sentence)

or

```suggestion
	// map of users per database suffix
```
(sentence fragment; see CONTRIBUTING.md)

or (the best option)

```suggestion
```
(since this commend doesn't add anything)